### PR TITLE
State: Remove unused memberships connected accounts state

### PR DESCRIPTION
**Pull request by tyxla**
#### Changes proposed in this Pull Request

* Follow-up to #47396, removes unused memberships connected accounts state as suggested in https://github.com/Automattic/wp-calypso/pull/47396#issuecomment-727842417

#### Testing instructions

* Verify nothing we're removing here is in use.

### CODE REVIEW

File: client/state/action-types.js [hunk 1]
```diff
@@ -587,10 +587,6 @@ export const MEDIA_SOURCE_CHANGE = 'MEDIA_SOURCE_CHANGE';
 export const MEDIA_SET_NEXT_PAGE_HANDLE = 'MEDIA_SET_NEXT_PAGE_HANDLE';
 export const MEDIA_SET_QUERY = 'MEDIA_SET_QUERY';
 export const MEDIA_CLEAR_SITE = 'MEDIA_CLEAR_SITE';
-export const MEMBERSHIPS_CONNECTED_ACCOUNTS_LIST = 'MEMBERSHIPS_CONNECTED_ACCOUNTS_LIST';
-export const MEMBERSHIPS_CONNECTED_ACCOUNTS_STRIPE_AUTHORIZE_REQUEST =
-	'MEMBERSHIPS_CONNECTED_ACCOUNTS_STRIPE_AUTHORIZE_REQUEST';
-export const MEMBERSHIPS_CONNECTED_ACCOUNTS_RECEIVE = 'MEMBERSHIPS_CONNECTED_ACCOUNTS_RECEIVE';
 export const MEMBERSHIPS_CONNECTED_ACCOUNTS_STRIPE_DISCONNECT_SUCCESS =
 	'MEMBERSHIPS_CONNECTED_ACCOUNTS_STRIPE_DISCONNECT_SUCCESS';
```
Comment (jsnajdr) [inline client/state/action-types.js:591]: This action could also be easily removed. It's identical to what `requestSettings` (`MEMBERSHIPS_SETTINGS`) does: refetch the membership settings for the site after Stripe was disconnected.

**Review by jsnajdr — APPROVED** [review]
Nice sleuthing :+1:

File: client/state/action-types.js (hunk 1 above)
Comment (tyxla) [reply client/state/action-types.js:591]: Nice find 👍  I'll follow-up in another PR to remove that one.

Comment (tyxla) [reply client/state/action-types.js:591]: See #47619 for removing the unnecessary action.

**Pull request merged**